### PR TITLE
kill keyboard shortcuts

### PR DESCRIPTION
**Pull request by orangejenny**
as discussed in #kill channel

@gcapalbo / @dannyroberts

### CODE REVIEW

**Comment by orangejenny** [comment]
Should delete https://help.commcarehq.org/display/commcarepublic/CommCareHQ+Keyboard+Shortcuts once this is deployed.

File: bower.json [hunk 1]
```diff
@@ -47,7 +47,6 @@
     "bootstrap": "3.3.6",
     "font-awesome": "fontawesome#4.5.0",
     "ui-select": "angular-ui/ui-select#v0.13.2",
-    "keymaster": "1.6.3",
```
Comment (proteusvacuum) [inline bower.json]: :key: :crown:

**Comment by proteusvacuum** [comment]
:heavy_minus_sign: 835

**Comment by dannyroberts** [comment]
Oh _hell_ yes!

**Comment by czue** [comment]
:hammer: :rage2: :skull:

**Comment by benrudolph** [comment]
:musical_keyboard: :gun:

**Comment by dannyroberts** [comment]
Should this also remove the toggle and the logic of when to enable keyboard shortcuts? I'm thinking of what happens when someone already has that toggle enabled. (I could just be missing where that is in the diff.)

**Comment by orangejenny** [comment]
@dannyroberts You mean the modal where you turn them on/off? It's deleted in bootstrap2/base.html and keyboard_shortcuts.html. The bindings get added by the calls to `key(...)`, which are all removed.

**Comment by dannyroberts** [comment]
Ah ok so there's no actual toggle, gotcha. Sounds good!

**Comment by dannyroberts** [comment]
:goberserk:

**Pull request merged**